data/language/en-GB.txt

```diff
@@ -3830,3 +3830,5 @@ STR_6788    :Sensitivity:
 STR_6789    :Analogue stick sensitivity multiplier
 STR_6790    :Deadzone: {COMMA32}%
 STR_6791    :Sensitivity: {COMMA32}%
+STR_6792    :Entrances
+STR_6793    :Peep profiles
```

Comment:
I'm not entirely convinced 'Peep profiles' is a good description for the contents, i.e. peep names and peep animation objects. Specifically, it's the word 'profile' that throws me off. Something more generic, like 'Guests and Staff', would be better, imo.